setup.py

```diff
@@ -62,14 +60,12 @@
     license='MIT license',
     zip_safe=False,
     keywords='azure iot edge dev tool',
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <3.8',
+    python_requires='>=3.6, <3.8',
```

Comment:
Why lock out 3.8?

Reply:
have not tested with 3.8 or 3.9. Will do this in a separate PR, as it is out of scope of this one.

Reply:
we have not tested 3.8 yet so cannot confirm support just yet.